### ISSUE TITLE
Добавить ссылку на логотип в админ панели

### ISSUE THREAD

**Issue by webinmd** [evolution-cms/evolution#1273]
В панели администратора переверстать верхнюю панель таким образом, чтобы можно было установить ссылку на логотип.

**Comment by Dmi3yy** [comment]
Что б в админке можно повесить свое лого ? или зачем не совсем понял

**Comment by webinmd** [comment]
> Что б в админке можно повесить свое лого ? или зачем не совсем понял

Нет, чтобы из любого места можно было быстро открыть еще одну вкладку с админкой просто кликнув колесиком (Открыть в новой вкладке) по логотипу. 
Лично я периодически пытаюсь нажать на логотип
